tslint.json

```diff
@@ -25,6 +39,18 @@
     "quotemark": [true, "single"],
     "semicolon": true,
     "triple-equals": true,
-    "variable-name": false
+    "variable-name": false,
+    "space-before-function-paren": [true, "never"],
+    "whitespace": [
+      true,
+      "check-branch",
+      "check-decl",
+      "check-module",
+      "check-operator",
+      "check-preblock",
+      "check-separator",
+      "check-type",
+      "check-typecast"
```

Comment:
Let's disable this, it looks more clear what the target is when there is no space after type casting.